### PR TITLE
Project Builder: Align workflow completeness radio buttons

### PR DESCRIPTION
**Pull request by eatyourgreens**
Align the workflow completeness radio buttons, in the lab, so that they are above one another.

Staging branch URL: https://pr-5834.pfe-preview.zooniverse.org

**before**
Screenshot showing the workflows table in the lab. The retirement count radio button is easy to miss because it isn't beneath the classification count radio button.

**after**
Screenshot showing the same table, but now the two radio buttons are vertically aligned.

# Required Manual Testing

- [ ] Does the non-logged in home page render correctly?
- [ ] Does the logged in home page render correctly?
- [ ] Does the projects page render correctly?
- [ ] Can you load project home pages?
- [ ] Can you load the classification page?
- [ ] Can you submit a classification?
- [ ] Does talk load correctly?
- [ ] Can you post a talk comment?

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] If the component is in coffeescript, is it converted to ES6? Is it free of eslint errors? Is the conversion its own commit?
- [ ] Are the tests passing locally and on Travis?

# Optional

- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you [resized and compressed](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) any image you've added?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?
- [ ] Have you followed the [Springer guidelines for commit messages](https://github.com/springernature/frontend-playbook/blob/master/git/git.md#commit-messages)?

### CODE REVIEW

**Review by beckyrother — APPROVED** [review]
So nice! Thanks for doing this Jim.

**Comment by eatyourgreens** [comment]
Github doesn't recognise Becky's approval as valid. 😞

File: app/pages/lab/workflows-table.jsx [hunk 1]
```diff
@@ -57,7 +57,7 @@ const WorkflowsTable = ({
                   Active
                 </label>
               </td>
-              <td>
+              <td class="stats_completeness_type">
```
Comment (srallen) [inline app/pages/lab/workflows-table.jsx:60]: Non-blocking, but we were using BEM convention last on PFE, so that'd still be useful I think.

**Review by srallen — APPROVED** [review]

File: app/pages/lab/workflows-table.jsx (hunk 1 above)
Comment (eatyourgreens) [reply app/pages/lab/workflows-table.jsx]: I took a guess at how PFE is using BEM and went for `standard-table__label__completeness` as the class name. Unfortunately, I don't think we documented how we were using BEM.

Comment (srallen) [reply app/pages/lab/workflows-table.jsx]: I'm not sure what would help regarding documentation, however, using the BEM standard, the block would be `standard-table`, the element is `label`, and `completeness` would be the modifier, so `standard-table__label--completeness` is how I'd do it.

**Pull request merged**